### DISCUSSION TITLE
How can I use useTranslate with new App Directory structure?

### DISCUSSION THREAD

**Discussion by tomfordrumm** [i18next/next-i18next#2242] [Q&A]
Hello!

I'm looking for the solution about how can u use useTranslate only. 

I see that Next 13 have a built in route Internationalization but it's not what I'm looking for. I don't want any language codes in the URL, I just need simple useTranslate. 

I see a lot of instructions and posts about how to do that with old `pages` structure, but have no luck to find about how to do that with new `app` structure. 

Any thought how can I reach my goal?

**Comment by adrai** [comment]
https://locize.com/blog/next-app-dir-i18n/
https://github.com/i18next/next-app-dir-i18next-example
https://github.com/i18next/next-app-dir-i18next-example-ts
https://github.com/i18next/next-app-dir-i18next-example/issues/12
